gradle/libs.versions.toml

```diff
@@ -1,30 +1,29 @@
 [versions]
-androidxCore = "1.12.0"
-androidxLifecycle = "2.7.0"
+androidxCore = "1.13.1"
+androidxLifecycle = "2.8.0"
```

Comment:
Nope: keep it at 2.7.0 for now https://www.reddit.com/r/androiddev/comments/1csjwne/lifecycle_280_only_compatible_with_compose_17/

Reply:
Ha, yes, I was pragmatic: it gave an error I didn't know how to fix, so I reverted.

Reply:
You can now use 2.8.1 hotfix https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.1

Reply:
Lifecycle is now on 2.8.2 with a supposedly final fix for the bug introduced in 2.8.0, better update before merging